data/block/metaBlock.go

```diff
@@ -523,3 +524,45 @@ func (m *MetaBlock) GetAdditionalData() headerVersionData.HeaderAdditionalData {
 	// no extra data for the initial version of meta block header
 	return nil
 }
+
+// CheckFieldsForNil checks a predefined set of fields for nil values
+func (m *MetaBlock) CheckFieldsForNil() error {
+	if m == nil {
+		return data.ErrNilPointerReceiver
+	}
+	if m.PrevHash == nil {
```

Comment:
PrevHash, PrevRandSeed and others fields checked here of the genesis block are not nil?

Reply:
they are not nil

Reply:
ok